react/src/pages/datasets/DatasetTable.tsx

```diff
@@ -114,6 +114,12 @@ export default function DatasetTable() {
             )}
             <MaterialTable
                 columns={[
+                    {
+                        title: "Dataset ID",
+                        field: "dataset_id",
+                        editable: "never",
+                        defaultFilter: paramFilter,
+                    },
```

Comment:
This is at the end of the row now. Did this happen in the merge process?

Reply:
I think so. I don't recall touching DatasetTable at all in this branch so I'll double check.